spark/interpreter/src/main/scala/org/apache/zeppelin/display/angular/AbstractAngularElem.scala

```diff
@@ -70,7 +70,7 @@ abstract class AbstractAngularElem(val interpreterContext: InterpreterContext,
     * @return
     */
   @ZeppelinApi
-  def model(name: String, value: Any): AbstractAngularElem = {
+  def model(name: String, value: AnyRef): AbstractAngularElem = {
```

Comment:
AnyRef is the same as object.
https://docs.scala-lang.org/tour/unified-types.html#scala-type-hierarchy